ArduCopter/GCS_Mavlink.cpp

```diff
@@ -583,7 +583,8 @@ static const ap_message STREAM_EXTRA3_msgs[] = {
 #endif
 };
 static const ap_message STREAM_PARAMS_msgs[] = {
-    MSG_NEXT_PARAM
+    MSG_NEXT_PARAM,
+    MSG_AVAILABLE_MODES
```

Comment:
What sets this rate, and did you consider making only send the mode on chance?

Reply:
You can request the full list be sent where it would use this rate. QGC actually just requests the modes one at a time, so the stream rate doesn't really mean anything, it would be the `SRx_PARAMS` parameter.